config/environments/development.rb

```diff
@@ -78,4 +78,13 @@
   # config.action_cable.disable_request_forgery_protection = true
   config.action_mailer.delivery_method = :letter_opener
   config.action_mailer.perform_deliveries = true
+  #config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+  :authentication => :plain,
+  :address => "smtp.eu.mailgun.org",
+  :port => 587,
+  :domain => "employeerp.co.uk",
+  :user_name => "valar@employeerp.co.uk",
+  :password => Rails.application.credentials.pw
+}
```

Comment:
this is not needed here as you are not sending anything through smtp right now in local development